deps/rabbitmq_management_agent/src/rabbit_mgmt_agent_sup_sup.erl

```diff
@@ -13,16 +13,40 @@
 -export([start_link/0, start_child/0]).
 
 -include_lib("rabbit_common/include/rabbit.hrl").
+-include("rabbit_mgmt_agent.hrl").
 
 start_child() ->
     supervisor2:start_child(?MODULE, sup()).
 
 sup() ->
-    {rabbit_mgmt_agent_sup, {rabbit_mgmt_agent_sup, start_link, []},
-     temporary, ?SUPERVISOR_WAIT, supervisor, [rabbit_mgmt_agent_sup]}.
+    #{
+        id      => rabbit_mgmt_agent_sup,
+        start   => {rabbit_mgmt_agent_sup, start_link, []},
+        restart => temporary,
+        wait    => ?SUPERVISOR_WAIT,
```

Comment:
The `supervisor2:child_spec()` type says this is called `shutdown`, not `wait`. Same problem in `rabbit_mgmt_sup_sup`.